lib/rules/no-else-return.js

```diff
@@ -88,6 +102,34 @@ module.exports = {
                     }
 
                     const endToken = sourceCode.getLastToken(node);
+
+                    /**
+                     * If else block includes block scope local variable declaration [let or const],
+                     * then it is not safe to remove else keyword [issue 11069]
```

Comment:
it's safe to remove the `else` keyword. it's not safe to remove the Block of the `else` branch